src/NServiceBus.Core.Tests/API/APIApprovals.ApproveNServiceBus.approved.txt

```diff
@@ -573,44 +570,28 @@ namespace NServiceBus
     {
         public static void EnableInstallers(this NServiceBus.BusConfiguration config, string username = null) { }
     }
-    public interface ISendOnlyBus : System.IDisposable
```

Comment:
some more missing obsoletes. Also there are obsolete messages redirecting to using ISendOnlyBus